### ISSUE TITLE
Check License does not take into account "Essential" objects

### ISSUE THREAD

**Issue by kmistrytnp** [DSaladinCH/al-object-helper#92]
**Version: 2.3.8**

**Describe the bug**
When reading the license file, the objects included as "Essentials Objects (hidden)" are not being included .  These are found in the "Module Objects and Permissions" part of the license.  These need to be taken into account in addition to the assigned objects ranges found in the "Object Assignment" section

Typically, this would include entries for Pages, Reports & XMLPorts 50000..50099 and Tables 50000..50009, and also codeunits 50000..50009 or 50000..50099

**To Reproduce**
1. Run the "AL Helper: Check License" option
2. Select your license txt file

**Expected behavior**
Include objects defined in the "Module Objects and Permissions" - in particular, those listed as "Essentials Objects (hidden)"

**Actual behavior**
Only objects ranges in the "Object Assignment" section are currently taken into account

**Comment by DominicSaladin** [comment]
Thank you for reporting this issue. I didn't include the Essentials objects, thats correct. I will change the implementation and include all objects that are included with the insert permission.

For example
```
Essentials Objects (hidden)                                 50000          50099          Codeunit            RIMDX
```

**Comment by DominicSaladin** [comment]
The fix will be released in version 2.3.9 today :)
